GETTINGSTARTED.md

```diff
@@ -122,3 +122,4 @@ If you are excited about Britecharts, want to add more configurable properties o
 [demos]: http://eventbrite.github.io/britecharts/tutorial-kitchen-sink.html
 [contribute]: https://github.com/eventbrite/britecharts/blob/master/CONTRIBUTING.md
 [home]: http://eventbrite.github.io/britecharts/
+[cdnDemo]: https://eventbrite.github.io/britecharts/tutorial-cdn.html
```

Comment:
"tutorial-cdn.html" should be deleted .https://github.com/eventbrite/britecharts/pull/272/commits/266d56260e1e59f2930b81ef65a7317c87a3f51c  
and please use this version named "cdn.html" which make it as a iframe page to make sure cdn static resources no conflict with layout.tmpl static resources ,also avoid conflict html elements.
https://github.com/eventbrite/britecharts/pull/272/commits/01bbcc55bd936a634481e9f0fe22770dc020d9d7#diff-5eed6d8287bc8ecf033a1662d37fd401